### PR TITLE
hack/usoc22: Update missing items in session 01

### PR DESCRIPTION
**Pull request by eduardvintila**
Signed-off-by: Eduard Vintilă <eduard.vintila47@gmail.com>

### CODE REVIEW

**Review by razvand — CHANGES_REQUESTED** [review]
Hi, @eduardvintila . Please make sure there is a blank line before and after each code block.

Remove `bash` as a snippet type for code block that use commands. We will only use it for scripts.

File: content/en/community/hackathons/sessions/baby-steps/content/Demos_00.-Manual-kraft-Installation.md [hunk 1]
```diff
@@ -9,7 +9,17 @@ $ sudo apt-get install -y --no-install-recommends build-essential \
 
 You'll also need QEMU for launching virtual/emulated machines which will run unikernels targeting the KVM platform:
 ```bash
-$ sudo apt-get -y install qemu-kvm qemu-system-x86
+$ sudo apt-get -y install qemu-kvm qemu-system-x86 sgabios
+```
+
+And the `bridge-utils` package for creating network bridges between the VM and the host system:
+```bash
```
Comment (RaduNichita) [inline content/en/community/hackathons/sessions/baby-steps/content/Demos_00.-Manual-kraft-Installation.md:16]:
```suggestion
```
```

File: content/en/community/hackathons/sessions/baby-steps/content/Demos_00.-Manual-kraft-Installation.md [hunk 2]
```diff
@@ -9,7 +9,17 @@ $ sudo apt-get install -y --no-install-recommends build-essential \
...
+$ sudo apt-get -y install bridge-utils
+```
+
+You might need to add `~/.local/bin` to your PATH:
+```bash
```
Comment (RaduNichita) [inline content/en/community/hackathons/sessions/baby-steps/content/Demos_00.-Manual-kraft-Installation.md:21]:
```suggestion
```
```

**Review by RaduNichita — CHANGES_REQUESTED** [review]

**Review by RaduNichita — CHANGES_REQUESTED** [review]
Check also the linter issues [1], please.

[1] https://github.com/eduardvintila/docs/actions/runs/2950165555/jobs/4714570245

**Comment by StefanJum** [comment]
Outdated by the USoC23 sessions.

**Pull request closed without merging**